### PR TITLE
Enable watcher tracking for all the pipelines

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds the jjb templates for all the pipelines part of the
PerfScale CI to the dynamic directory for watcher to be able to 
track/make changes. This way everything is automated. Any change 
to the default workloads enabled and the properties files can be made 
through github similar to how it's done for the workloads.

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
LGTM, I just reviewed the one pipeline, i assume they are all copy and pasta. ;)

**Review by whitleykeith — APPROVED** [review]
lgtm

File: jjb/dynamic/scale-ci-pipeline-4.7.yml [hunk 1]
```diff
@@ -0,0 +1,337 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    concurrent: true
+    description: Collection of Jenkins jobs to install, configure, scaleup ocp and
+      run various performance and scale tests on OpenShift across various public and
+      private clouds. <!-- Managed by Jenkins Job Builder -->
+    disabled: false
+    name: scale-ci-pipeline-4.7
```
Comment (mohit-sheth) [inline jjb/dynamic/scale-ci-pipeline-4.7.yml:9]: can we have pipeline names that match current names i.e all caps
same for the other pipelines too

Comment (chaitanyaenr) [reply jjb/dynamic/scale-ci-pipeline-4.7.yml]: Fixed it, thanks.

Comment (chaitanyaenr) [reply jjb/dynamic/scale-ci-pipeline-4.7.yml]: You are right about the names @mohit-sheth, there might be something wrong with the xml files, changing the names to use uppercase.

Comment (chaitanyaenr) [reply jjb/dynamic/scale-ci-pipeline-4.7.yml]: Done.

**Review by amitsagtani97 — APPROVED** [review]
LGTM!

**Pull request merged**